src/Factory/Api/Mastodon/Status.php

```diff
@@ -78,10 +78,10 @@ public function __construct(LoggerInterface $logger, Database $dba,
 	 * @throws HTTPException\InternalServerErrorException
 	 * @throws ImagickException|HTTPException\NotFoundException
 	 */
-	public function createFromUriId(int $uriId, $uid = 0): \Friendica\Object\Api\Mastodon\Status
+	public function createFromUriId(int $uriId, $uid = 0, $item = []): \Friendica\Object\Api\Mastodon\Status
```

Comment:
Please explain and document the `$item` parameter since it is immediately overridden on line 85.

Reply:
That's a leftover that was part of my tests.